### DISCUSSION TITLE
The php method 'ucwords' should be added to Str facade.

### DISCUSSION THREAD

**Discussion by yogeshgalav** [laravel/framework#46722] [Ideas] [answered]
Like Str::ucfirst I think we should also add Str::ucWords as a substitute for php method ucwords.
Also if possible rename Str::ucfirst to Str::ucFirst.

**Comment by juse-less** [comment, marked as answer]
There was a similar question a while ago, as well.
See my answer here: https://github.com/laravel/framework/discussions/44268#discussioncomment-3716257. 🙂

But the gist is [`Str::title()`](https://laravel.com/docs/9.x/helpers#method-title-case).
